src/util/kaldi-table-inl.h

```diff
@@ -88,8 +101,10 @@ template<class Holder>  class SequentialTableReaderScriptImpl:
 
   SequentialTableReaderScriptImpl(): state_(kUninitialized) { }
 
+  // you may call Open from states kUninitialized and kError.
```

Comment:
Do we require all comments be capitalized as proper sentences? Google style does, IIRC.